services/controlplane.go

```diff
@@ -20,17 +20,17 @@ func RunControlPlane(controlHosts, etcdHosts []*hosts.Host, controlServices v3.R
 			return err
 		}
 		// run kubeapi
-		err := runKubeAPI(host, etcdHosts, controlServices.KubeAPI, authorizationMode)
+		err := runKubeAPI(host, etcdHosts, controlServices.KubeAPI, authorizationMode, df)
```

Comment:
I don't want to sound as a syntax nazi, but can we use more descriptive name instead of df? May be dFactory?